store/src/test/java/com/nytimes/android/external/store3/CharsetTest.kt

```diff
@@ -0,0 +1,38 @@
+package com.nytimes.android.external.store3
```

Comment:
Why would you need to test `Charset` class? What i mean is that it is provided by Java and should supposedly be tested. You usually do not need to test third party dependencies, only the integration with them :) I would suggest removing this test!

Reply:
good point! removed!